pkg/awsutils/awsutils.go

```diff
@@ -1922,16 +1938,16 @@ func (cache *EC2InstanceMetadataCache) getENIsFromPaginatedDescribeNetworkInterf
 	return innerErr
 }
 
-// SetCNIUnmanagedENIs Set unmanaged ENI set
-func (cache *EC2InstanceMetadataCache) SetCNIUnmanagedENIs(eniID []string) error {
+// SetMultiCardENIs Set unmanaged ENI set
+func (cache *EC2InstanceMetadataCache) SetMultiCardENIs(eniID []string) error {
 	if len(eniID) != 0 {
 		cache.cniunmanagedENIs.Set(eniID)
```

Comment:
nit: while we are here, let's update this `cniunmanagedENIs` variable name to be `multiCardENIs`